client/src/components/Login/Login.css

```diff
@@ -0,0 +1,82 @@
+.sign-up-form {
+  display: flex;
+  width: 70%;
+  height: 360px;
+  flex-direction: column;
+  background-color: #f7f7f7;
+  padding-bottom: 10px;
+}
+
+.sign-up-form__button-container {
+  margin: 0% 0% 0% 35px;
+  display: flex;
+  align-content: center;
+  width: 95%;
+}
+
+.sign-up-form__text {
+  margin: 30px 0% 12px 0%;
+  display: flex;
+  flex-direction: column;
+  background-color: inherit;
+  border-top: 0px;
+  border-right: 0px;
+  border-left: 0px;
+  border-color: #e5e5e5;
+  width: 95%;
+}
+
+.sign-up-form__text:focus {
+  border-bottom: 2px solid goldenrod;
+  outline: none;
+}
+
+.sign-up-form__label {
+  margin: 0% 0% 6px 35px;
+  font-size: 13px;
+}
+
+.sign-up-form__header {
+  margin: 35px 5px 35px 31px;
+  font-size: 17px;
+}
+
+.sign-up-form__anchor {
+  font-size: 13px;
+  margin: 0px 0px 25px 35px;
+}
+
+.sign-up-form__checkbox {
+  margin: 15px 0px 0px 20px;
+}
+
+.sign-up-form__submit {
+  background-color: inherit;
+  border: 2px solid #e5e5e5;
+  padding: 10px 13px;
+}
+
+.sign-up-form__submit:focus {
+  outline: none;
+}
+
+.sign-up-form__submit:hover {
+  border: 2px solid goldenrod;
+  color: goldenrod;
+  transition: 1s;
+}
+
+.remember-me__span {
+  font-size: 13px;
+  margin: 15px 0% 0% 6px;
+}
+
+@media only screen and (min-width: 1000px) {
```

Comment:
My opinion is that we should skip media query in this project, it takes a lot of time, and I don't think it is priority

@amusameh , @FarahZaqout 
what do you think?

Reply:
I am inclined to agree. I think we should make a judgment as to how quick we can progress with or without it. Surely, CSS is not the priority in this project. I don't mind skipping it.